examples/controls/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "webpack-exhibit",
```

Comment:
Delete name/version/description/author etc - goal is to have as few lines of config as possible